index.bs

```diff
@@ -5930,6 +5930,21 @@ H(z) = \frac{\frac{b_0}{a_0} + \frac{b_1}{a_0}z^{-1} + \frac{b_2}{a_0}z^{-2}}
 $$
 </pre>
 
+Transfer functions work as \(Y(z) = H(z) X(z)\), where \(Y(z), X(z)\) are the
+<a href="https://en.wikipedia.org/wiki/Laplace_transform">Laplace transforms</a>
+of the input and output respectively.
+
+This <a href="https://en.wikipedia.org/wiki/Infinite_impulse_response#Transfer_function_derivation">
+is equivalent</a> to the following time-domain equation:
+
+<pre nohighlight>
+$$
+y[n] = \frac{b_0}{a_0} x[n] + \frac{b_1}{a_0} x[n - 1] + \frac{b_2}{a_0} x[n - 2]
```

Comment:
This notation doesn't match the time-domain formula for the IIR filter.  Make it consistent.  Actually, I think I'd just copy the IIR result, but expand it out with N = 2 and M = 2.

Reply:
I feel like the IIR filter should either use subscripts or square brackets, `()` for indexing discrete data is not as common IME

Reply:
This probably depends on the textbooks you're familiar with, and it's very common in Matlab. :-)

Reply:
Fixed